src/components/pill/mixin/index.js

```diff
@@ -0,0 +1,41 @@
+import { vlElement } from '../../../utils/core';
```

Comment:
Deze mixin wordt nu enkel door de pillbutton gebruikt. Om verwarring te vermijden zou ik dit gewoon in de pill-button component steken.

Reply:
Fixed.